Dockerfile

```diff
@@ -0,0 +1,35 @@
+# This Dockerfile builds an image containing the Mac and Windows version of kube-compare
+# layered on top of the Linux cli image.
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-builder-multi-openshift-4.17 AS builder-rhel-8
+WORKDIR /go/src/github.com/openshift/kube-compare
+COPY . .
+RUN go work vendor
+RUN make cross-build --warn-undefined-variables
+
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-builder-multi-openshift-4.17 AS builder-rhel-9
```

Comment:
`rhel-8-golang-1.22-builder-multi-openshift-4.17` should be rhel9 right?

Reply:
oops, good catch